ControlLaptop/Examples/Log-windtunnel-influx/log-windtunnel-influx.py

```diff
@@ -0,0 +1,164 @@
+# Example for logging data to influxdb using python
+# Sourced from the influxdb python docs
+
+import random
+import time
+from datetime import datetime 
+
+from influxdb import InfluxDBClient
+
+# Example data payload
+# Tags contain metadata about the measurement - always a string
+# Thinking we could have something like below for each measurement
+
+def main():
+        
+    telemetry = [
+           {
+            "measurement": "temperature",
+            "tags": {
+                "host": "server01",
+                "region": "us-west"
+            },
+            #"time": f"{datetime.now()}",
+            "fields": {
+                "Celsius": 1.0,
+                "Fahrenheit": 3,
+               }
+           },
+        
+          {
+            "measurement": "pressure",
+            "tags": {
+                "host": "server01",
+                "region": "us-west",
+                "config": "1",
+            },
+            "fields": {
+                "millibar": 1.0,
+                "Pascal": 3.0,
+                }
+           },
+         
+         {
+            "measurement": "Height",
+            "tags": {
+                "host": "server01",
+                "region": "us-west",
+                "config": "1",
+            },
+            "fields": { 
+                "Metre": 1.0,
+                "Feet": 3.0,
+                }
+        },
```

Comment:
So if you look at the indentation of this bracket compared to the others, you'll see that it appears that they are not indented consistently. It would be nicer if they are all indented the same amount.